src/database/queries/bulkUploads/selectById.sql

```diff
@@ -3,6 +3,7 @@ SELECT
   file_name as "fileName",
   source_key as "sourceKey",
   status as "status",
+	file_size as "fileSize",
```

Comment:
It looks like this is tab-indented while the surrounding SQL is space-indented.
```suggestion
  file_size as "fileSize",
```